pkg/machinery/meta/meta.go

```diff
@@ -82,3 +123,11 @@ func DecodeValues(s string) (Values, error) {
 
 	return result, nil
 }
+
+func hasGzipMagic(b []byte) bool {
+	if len(b) < 2 {
+		return false
+	}
+
+	return b[0] == 0x1f && b[1] == 0x8b
```

Comment:
maybe a comment here from the spec?

Reply:
what kind of comment are you looking for?

Reply:
it's gzip magic number as the function name suggests!

https://cs.opensource.google/go/go/+/refs/tags/go1.21.6:src/compress/gzip/gunzip.go;l=20-21

Reply:
but I see your point